src/keystore.h

```diff
@@ -44,11 +45,21 @@ class CKeyStore
     virtual bool RemoveWatchOnly(const CScript &dest) =0;
     virtual bool HaveWatchOnly(const CScript &dest) const =0;
     virtual bool HaveWatchOnly() const =0;
+
+    //! Add a spending key to the store.
+    virtual bool AddSpendingKeyPaymentAddress(const libzcash::SpendingKey &key, const libzcash::PaymentAddress &address) =0;
```

Comment:
The implementation seems to associate the key with the inconsistent address, which seems wrong. Consider removing this method.